api/grpc_server.go

```diff
@@ -46,6 +47,8 @@ type SpacemeshGrpcService struct {
 	Logging       LoggingAPI
 }
 
+var _ pb.SpacemeshServiceServer = &SpacemeshGrpcService{}
```

Comment:
if this is a type check you can achieve this without allocating. try (*pb.SpacemeshServiceServer)(nil)